packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -0,0 +1,739 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { IRandom, makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { benchmark, BenchmarkType } from "@fluid-tools/benchmark";
+import {
+	createField,
+	FieldKinds,
+	getField,
+	isUnwrappedNode,
+	namedTreeSchema,
+	singleTextCursor,
+} from "../../feature-libraries";
+import { brand, unreachableCase } from "../../util";
+import { ITestTreeProvider, TestTreeProvider } from "../utils";
+import { ISharedTree } from "../../shared-tree";
+import {
+	schemaMap,
+	// eslint-disable-next-line import/no-internal-modules
+} from "../feature-libraries/editable-tree/mockData";
+import {
+	FieldKindIdentifier,
+	fieldSchema,
+	GlobalFieldKey,
+	JsonableTree,
+	LocalFieldKey,
+	moveToDetachedField,
+	rootFieldKey,
+	rootFieldKeySymbol,
+	SchemaData,
+	TransactionResult,
+	TreeSchemaIdentifier,
+	UpPath,
+	ValueSchema,
+} from "../../core";
+
+enum TreeShape {
```

Comment:
It would be great to document eaxtly what these shapes mean, and why they are the shapes we are testing with here

Reply:
added documentation clarifying what the shapes mean

Reply:
It would be better to put the docs for each shape on the shapes themselves.
It would also be good to include why we are using these shapes.

Something like:
```
/**
 * Shapes of trees to test with.
 */
enum TreeShape {
	/**
	 * Each node is inserted directly 1 layer below the root node, making a wide tree.
	 * Used to measure the performance of editing long sequences.
	 */
	Wide = 0,
	/**
	 * Each node is inserted one layer below the previous node, making a stick-like tree.
	 * Used to measure the performance costs related to the length of paths.
	 */
	Deep = 1,
}

```

Reply:
Updated documentation